CONTRIBUTING.md

```diff
@@ -200,15 +200,15 @@ Runtime guarded features may either set true (running the new code by default) i
 after a testing interval, or during the next release cycle, at the PR author's and reviewing
 maintainer's discretion. Generally all runtime guarded features will be set true when a
 release is cut. Old code paths for refactors can be cleaned up after a release and there has been
-some production run time. Old code for behavioral changes will be deprecated after six months.
+some production run-time. Old code for behavioral changes will be deprecated after six months.
```

Comment:
Runtime?

Reply:
Most changes I made similar to this and a lot of "setup"/ "set-up"'s you'll see are adjustments for when the word is being used as a noun or as a verb. Generally, nouns will be hyphenated and verbs will be written out as two separate words. Apologies if it seems a bit nit-picky, I just figured I'd try and make the writing as consistent as possible.